src/swagger/api.yaml

```diff
@@ -7346,6 +7346,7 @@ definitions:
       - SAML_2_0
       - OPENID_CONNECT
       - SAML_1_1
+      - CUSTOM
```

Comment:
This needs to be in the official spec first:
https://github.com/okta/okta-management-openapi-spec/blob/master/dist/spec.yaml#L7323-L7336